docs/Primer/Classes.md

```diff
@@ -0,0 +1,221 @@
+!!! note
+    **Class** - A cohesive package that consists of a particular kind of compile-time metadata. A class describes the rules by which objects behave. A class specifies the structure of data which each instance contains as well as the methods (functions) which manipulate the data of the object.
```

Comment:
>A cohesive package that consists of a particular kind of compile-time metadata.

That's a pretty awkward definition for a class, especially in an introductory primer-level document.  Heck, I've been doing OOP for a couple decades and I have no idea what that sentence means! :P